ACHNBrowserUI/ACHNBrowserUI/packages/Backend/Sources/Backend/environments/UserCollection.swift

```diff
@@ -260,8 +260,8 @@ public class UserCollection: ObservableObject {
         save()
     }
 
-    public func deleteChore(at index: Int) {
-        chores.remove(at: index)
+    public func deleteChore(_ chore: Chore) {
+        chores.removeAll { $0.id == chore.id }
```

Comment:
`Chore` is `Equatable` so you can directly compare them
```suggestion
        chores.removeAll { $0 == chore }
```

Reply:
Done. Thanks @renaudjenny!